packages/jellyfish-api-core/__tests__/category/poolpair/compositeSwap.test.ts

```diff
@@ -118,6 +118,50 @@ describe('compositeSwap', () => {
     await testing.container.stop()
   })
 
+  it('should compositeSwap 1 sat', async () => {
```

Comment:
check for dex fee burn. should be 0 in this case
https://github.com/DeFiCh/ain/blob/c0b54230b2a31119a9d2dfdb4689282866625097/src/masternodes/mn_checks.cpp#L3881
Also please add a burn check for the `should compositeSwap` test as well.